### ISSUE TITLE
Repeated updateConfig causes "velocity" change

### ISSUE THREAD

**Issue by SmitdCatchy** [3d-dice/dice-box#101]
@frankieali If I want to roll different amount of dice each time and rescale them to fit the canvas area and I call updateConfig() the dice seem to be getting "quicker" to the point when they are starting to "twitch" on landing. This behavior appears after a fe hundred dice being rolled consistently for me.

**Comment by frankieali** [comment]
I'll look into it. Can you give me a few more details on how to recreate this issue?
The `throwForce` and `mass` of the physics colliders are tied to `scale` setting so there might be a math issue that's adding up over time. Dice twitching might be associated with mass eventually dropping to zero.

**Comment by SmitdCatchy** [comment]
Many thanks! Sure I can.

So basically in the application there are different characters which roll
different type and number of dice that is why I have to change the number
and scale of the dice.

I created a service which initializes the dicebox and before each call
check the dice type and number and calls the updateConfig with the new
scale value, the mass and other attributes are untouched on my part.

But after nearly 10 rolls of many dice this velocity issue occures.

Sadly, I cannot share the codebase with you, but will ask about a code
snippet from the service if it is any help.

**Comment by frankieali** [comment]
Looks like you can get around this defect by setting the `throwForce` config option when you first initialize the DiceBox
```JavaScript
let Box = new DiceBox("#dice-box", {
  assetPath: "assets/",
  origin: "https://unpkg.com/@3d-dice/dice-box@1.0.12/dist/",
  scale: 6,
  throwForce: 5,
});
```
There seems to be a mixup as to what value should be used as the base value when calculating throwForce. Setting it on initialization keeps the base value stable. Not setting the `throwForce` value means the previous computed value gets recomputed, slowly making the value smaller.

**Comment by frankieali** [comment]
Fixed with release of 1.0.13

**Comment by SmitdCatchy** [comment]
Awesome, thank you so much!

**Comment by frankieali** [comment]
I just noticed that I accidentally introduced a bug with the fix in 1.0.13. It was related to scale and how the config options might have been updated with undefined values. I just posted version 1.0.14 to fix this.

**Comment by SmitdCatchy** [comment]
Thank you for the heads up!
